trunk/apps/dir2hdf5.py

```diff
@@ -0,0 +1,118 @@
+
+import argparse
+import sys
+import os
+import os.path as osp
+
+import h5py
+import numpy as np
+
+from nn.profiler import setupLogging
+from dataset.reader import readImage
+
+STORAGE_TYPE = np.uint8
+
+def dir2hdf5(directory, outputfile, convert, log):
```

Comment:
I believe these methods would be better in dataset package somewhere. I'm open to suggestions, however they could easily go in hdf5.py